pgjdbc/src/test/java/org/postgresql/test/jdbc42/DatabaseMetaDataTest.java

```diff
@@ -79,6 +79,16 @@ void getColumnsForNullScale() throws Exception {
     assertFalse(rs.next());
   }
 
+  @Test
+  void getColumnsForSchema() throws Exception {
+    DatabaseMetaData dbmd = conn.getMetaData();
+
+    ResultSet rs = dbmd.getColumns(null, "%", "decimaltest", "%");
+    assertTrue(rs.next());
+    assertEquals("a", rs.getString("COLUMN_NAME"));
+    assertEquals(0, rs.getInt("DECIMAL_DIGITS"));
```

Comment:
Can we add a verification for the `current_database` column name?

Reply:
I have added it to the test. Notice that `getColumns` reports the current database in the `TABLE_CAT` column: https://github.com/pgjdbc/pgjdbc/blob/master/pgjdbc/src/main/java/org/postgresql/jdbc/PgDatabaseMetaData.java#L1671

Reply:
Yes, I had this in mind as well.

Reply:
Latest commit is removing `current_database()` where possible and use either the argument or use the connection's catalog name.